amethyst_renderer/src/pipe/stage.rs

```diff
@@ -38,16 +38,29 @@ pub struct Stage {
     clear_color: Option<[f32; 4]>,
     clear_depth: Option<f32>,
     enabled: bool,
-    passes: Vec<CompiledPass>,
+	/// Draw passes of the draw stage
+    pub passes: Vec<CompiledPass>,
```

Comment:
The indentation on this comment looks funny, can you fix that?

Reply:
For some reason the comment is aligned on my side :/

Reply:
Make sure to use spaces, we don't use tab characters in this project.